provider/provider.go

```diff
@@ -2,6 +2,7 @@ package provider
 
 import (
 	"context"
+	"github.com/coder/terraform-provider-coder/v2/provider/helpers"
```

Comment:
nit: ordering

Reply:
Addressed in https://github.com/coder/terraform-provider-coder/pull/421/commits/de3cf097faa71bf1512cfb325e9d4dd39d0999a2